### PR TITLE
[UPDATE BONUS GUIDE] Fulcrum: Add TCP

### PR DESCRIPTION
**Pull request by DanielPX1**
#### What

Add TCP as default for Fulcrum users

### Why

More ways to connect to Fulcrum server. Issues have been found for example on BlueWallet, when using SLL + Tor. 

#### How

Updating guide

#### Scope

- [ ] review

### CODE REVIEW

File: guide/bonus/bitcoin/fulcrum.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Table of contents
 
 ---
 
-Fulcrum is a replacement for an Electrs, these two services cannot be run at the same time (due to the same standard ports used)
+Fulcrum is a replacement for an Electrs, these two services cannot be run at the same time (due to the same standard ports used). Performance issues have been found on Raspberry Pi 4GB, it is recommended to install Fulcrum on 8GB version.
```
Comment (twofaktor) [inline guide/bonus/bitcoin/fulcrum.md:43]: Change to -> 

Fulcrum is a replacement for an Electrs, these two services cannot be run at the same time (due to the same standard ports used), remember to stop Electrs doing "sudo systemctl stop electrs". Performance issues have been found on Raspberry Pi 4GB, it is recommended to install Fulcrum on 8GB RAM version.

**Review by twofaktor — CHANGES_REQUESTED** [review]

**Comment by twofaktor** [comment]
Recover comment about TCP connection in this [line](https://github.com/raspibolt/raspibolt/pull/1197/files#diff-730f9901c4e7cac9c3b645e8bbe633d5e208fa99886bbee3fc28d2c873c3b8a0R260): 

`RaspiBolt uses SSL as default for Fulcrum, but some wallets like [BlueWallet](https://bluewallet.io/) do not support SSL over Tor. You may as well need to use TCP for other reasons.`

File: guide/bonus/bitcoin/fulcrum.md (hunk 1 above)
Comment (DanielPX1) [reply guide/bonus/bitcoin/fulcrum.md]: Resolved

**Comment by DanielPX1** [comment]
Not sure I understood what line you mean, but I have put it at the start of a guide @twofaktor 

> Recover comment about TCP connection in this [line](https://github.com/raspibolt/raspibolt/pull/1197/files#diff-730f9901c4e7cac9c3b645e8bbe633d5e208fa99886bbee3fc28d2c873c3b8a0R260):
> 
> `RaspiBolt uses SSL as default for Fulcrum, but some wallets like [BlueWallet](https://bluewallet.io/) do not support SSL over Tor. You may as well need to use TCP for other reasons.`

**Comment by twofaktor** [comment]
> Not sure I understood what line you mean, but I have put it at the start of a guide @twofaktor
> 
> > Recover comment about TCP connection in this [line](https://github.com/raspibolt/raspibolt/pull/1197/files#diff-730f9901c4e7cac9c3b645e8bbe633d5e208fa99886bbee3fc28d2c873c3b8a0R260):
> > `RaspiBolt uses SSL as default for Fulcrum, but some wallets like [BlueWallet](https://bluewallet.io/) do not support SSL over Tor. You may as well need to use TCP for other reasons.`

Line 262, but is fine anyway

tACK!

**Review by twofaktor — APPROVED** [review]

**Comment by DanielPX1** [comment]
Great!

**Comment by DanielPX1** [comment]
Did the line 262 in the end, it looks much better. I added one more sentence as well. 

@twofaktor

**Review by twofaktor — APPROVED** [review]
tACK! Does some other maintainer want to review this?

**Comment by twofaktor** [comment]
Let's go to the master! 🚀

**Pull request merged**